PoGo.Necrobot.Window/MainClientWindow.xaml.cs

```diff
@@ -63,9 +63,9 @@ public partial class MainClientWindow : MetroWindow
                 { LogLevel.New, new Tuple<string, string, string>("Blue", "#0883FF", "#0883FF") },
                 { LogLevel.SoftBan, new Tuple<string, string, string>("Pink", "#FF1E8E", "#EC008C") },
                 { LogLevel.LevelUp, new Tuple<string, string, string>("Blue", "#0883FF", "#0883FF") },
+                { LogLevel.BotStats, new Tuple<string, string, string>("Blue", "#0883FF", "#0883FF") },
                 { LogLevel.Gym, new Tuple<string, string, string>("Purple", "#8308FF", "#197B30") },
-                { LogLevel.Service , new Tuple<string, string, string>("Blue", "#0883FF", "#0883FF") },
-                { LogLevel.BotStats , new Tuple<string, string, string>("Cyan", "#B4E1FD", "#EC008C") }
+                { LogLevel.Service , new Tuple<string, string, string>("Blue", "#0883FF", "#0883FF") }
```

Comment:
LogLevel.BotStats  ->	Fixes bug v1.0.0.178 Exception caught, writing LogBuffer (color changed cyan to blue) (missid value in x.178) fixed in x.179 and change color now